.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "testfiles"]
+	path = testfiles
+	url = https://github.com/gimli-rs/object-testfiles
```

Comment:
I don't know how big this repo will be, but when you use a git dep with cargo, it will recursively clone all submodules. This may be something to keep in mind.